content/ytt/docs/latest/faq.md

```diff
@@ -60,7 +60,9 @@ Additional resources: [overlay remove docs](lang-ref-ytt-overlay.md#overlayremov
 
 ## How do I add items to an existing array?  
 
-The default merging behaviour for arrays is to overwrite them, as there isn't any default matching criteria (unlike maps which use their keys). To add an item, either provide that matching annotation (eg. `#@overlay/match by="field_name"`), or use the `#@overlay/append` annotation to add to the end of the list. Note that the append annotation must be applied to each item you want to insert.
+Using v0.32.0 or later, the default behavior of overlays is to append array items. Simply put your array item in an overlay. 
+
+Prior to v0.32.0, the default merging behaviour for arrays is to overwrite them, as there isn't any default matching criteria (unlike maps which use their keys). To add an item, either provide that matching annotation (eg. `#@overlay/match by="field_name"`), or use the `#@overlay/append` annotation to add to the end of the list. Note that the append annotation must be applied to each item you want to insert.
```

Comment:
> default merging behaviour for arrays is to overwrite them

there was no default merging behaviour. one had to explicitly specify overlay/match if overlay/merge method was being used.

Reply:
This is to addressed in the new PR here: https://github.com/vmware-tanzu/carvel.dev/pull/64/commits/d5dde75e954de0d0ad4cc9bc5a4cf0a318403807